infra/ci/sandbox/Dockerfile

```diff
@@ -55,16 +51,41 @@ RUN set -ex; \
                xdg-utils fonts-liberation fonts-ipafont-gothic fonts-wqy-zenhei \
                fonts-thai-tlwg fonts-kacst fonts-freefont-ttf
 
-# Cleanup to reduce image size
-RUN     apt-get -y autoremove; \
-        rm -rf /var/lib/apt/lists/* /usr/share/man/* /usr/share/doc/*; \
-        rm -rf /root/.cache/;
+RUN set -ex; \
+    export DEBIAN_FRONTEND=noninteractive; \
+    apt-get -y install \
+        procps jq python3-jwt python3-requests python3 python3-oauth2client \
+        python3-httplib2 python3-google-auth \
+        python3-googleapi
 
-COPY testrunner.sh /ci/testrunner.sh
-COPY init.sh /ci/init.sh
-RUN chmod -R a+rx /ci/
 
-VOLUME [ "/ci/cache", "/ci/ramdisk", "/ci/artifacts" ]
-ENTRYPOINT [ "tini", "-g", "--" ]
-CMD [ "bash", "/ci/init.sh" ]
+RUN set -ex; \
+    groupadd -g 1986 perfetto; \
+    useradd -m -d /opt/ci -u 1986 -g perfetto perfetto; \
+    apt-get -y autoremove; \
+    rm -rf /root/.cache/ /usr/share/man/* /usr/share/doc/*;
+
+WORKDIR /opt/ci
 
+COPY sandbox_entrypoint.sh ./
+COPY artifacts_uploader.py ./
+COPY tmp/config.py ./
+
+# Download GitHub Actions runner.
+# We should upkeep GHAR_VER with the latest version in
```

Comment:
are we planning on doing this in a followup or just do it as and when necessary?

Reply:
Yeah this needs to be some upkeep we do once/twice per year.
Can't think of a better option tbh